core/src/main/scala/org/ensime/core/RichPresentationCompiler.scala

```diff
@@ -145,21 +145,6 @@ trait RichCompilerControl extends CompilerControl with RefactoringControl with C
   def askReloadAndTypeFiles(files: Iterable[SourceFile]) =
     askOption(reloadAndTypeFiles(files))
 
-  def askUsesOfSymAtPos(pos: Position)(implicit ec: ExecutionContext): Future[List[RangePosition]] = {
```

Comment:
consider the case where the user has added a reference in their currentyl open file, but not yet compiled that file.